versioned_docs/version-8.2/components/modeler/web-modeler/play-your-process.md

```diff
@@ -22,7 +22,7 @@ There are four different main views in Play:
 Click on the action icons next to a BPMN element to play through the process. Depending on the BPMN element, there may be a different action:
 
 - **User tasks** display the defined form on click, if possible. Form results are saved for the rest of the session and auto-filled. You can click `Reset` to reset the form to its defaults.
-- **Service tasks** are simulated on click. You can mock an endpoint by manually entering variables when completing a service task
+- **Service tasks** are simulated on click. You can mock a cluster URL by manually entering variables when completing a service task
```

Comment:
```suggestion
- **Service tasks** are simulated on click. You can mock a service by manually entering variables when completing a service task
```

Reply:
@Zelldon - WDYT on this case. The cluster URL is not mocked right? Just the response of the service.

Reply:
Yeah I think the replacement here doesn't make sense.

Reply:
Resolved in latest commit 👍